libcentrifugo/application.go

```diff
@@ -341,16 +341,11 @@ func (app *Application) pubControl(method string, params []byte) error {
 	app.RLock()
 	controlChannel := app.config.ControlChannel
 	app.RUnlock()
-	return app.engine.publish(controlChannel, messageBytes, nil)
+	return app.enginePublish(controlChannel, messageBytes, nil)
 }
 
-// pubAdmin publishes message into admin channel so all running
-// nodes will receive it and send to admins connected.
-func (app *Application) pubAdmin(message []byte) error {
-	app.RLock()
-	adminChannel := app.config.AdminChannel
-	app.RUnlock()
-	return app.engine.publish(adminChannel, message, nil)
+func (app *Application) enginePublish(chID ChannelID, message []byte, opts *publishOpts) error {
```

Comment:
Nitpick but that name is a little bit confusing. It's real purpose is a synchronous helper so maybe `publishSync` would be cleaner name?

Not a big deal if you disagree though.